src/inject/dynamic-theme/variables.ts

```diff
@@ -644,7 +644,7 @@ function parseRawValue(color: string) {
         splitted.forEach((number) => {
             resultInRGB += `${number.trim()}, `;
         });
-        resultInRGB = resultInRGB.substr(0, resultInRGB.length - 2);
+        resultInRGB = resultInRGB.slice(0, resultInRGB.length - 2);
```

Comment:
Personally I prefer using `substring` method for Strings and `slice` for Arrays.

Reply:
I like `slice()` more as it's [stricter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#differences_between_substring_and_slice) but your repo, your rules. I will change it to `substring()`